codegen.ts

```diff
@@ -1,32 +1,116 @@
 import { CodegenConfig } from '@graphql-codegen/cli';
 
-import { fetchConfig } from './src/lib/fetchConfig';
+import { fetchConfig, fetchConfigSanity } from './src/lib/fetchConfig';
```

Comment:
Wouldn't it be possible to use a separate config file for sanity and contentful? You might need to add to package.json to run both scripts.